unity-client/Assets/Scripts/MainScripts/DCL/Helpers/NFTHelper/NFTMarkets/OpenSea_Internal/Request.cs

```diff
@@ -127,28 +143,39 @@ IEnumerator Fetch(float delayRequest)
             {
                 using (UnityWebRequest request = UnityWebRequest.Get(url))
                 {
-                    if (OpenSeaRequestController.VERBOSE) Debug.Log($"RequestGroup: Request to OpenSea {url}");
+                    if (OpenSeaRequestController.VERBOSE)
+                        Debug.Log($"RequestGroup: Request to OpenSea {url}");
                     yield return request.SendWebRequest();
 
                     AssetsResponse response = null;
 
                     if (!request.isNetworkError && !request.isHttpError)
                     {
+                        Debug.Log(request.downloadHandler.text);
```

Comment:
Shouldn't we put this log inside a VERBOSE conditional?

Reply:
This shouldnt have been commited hahaha, nice catch!